### PR TITLE
Various cleanup

### PR DESCRIPTION
**Pull request by ThatIsAUsername**
I changed a bunch of stuff. Tell me if any of it is objectionable. You might have an easier time going a commit at a time, since a lot of the changes are mostly unrelated.

If there is any additional cleanup you want, this is a good potpourri branch to shoehorn stuff into.

One thing we've talked about that I did not change was unit names: Infantry->Rifles, Mech->RPG, Tank->ScoutTank, MdTank->Tank/BattleTank. Not sure if I really want to go changing things since they are recognizable as is (though if I introduce some kind of mechwarrior unit, it coulndn't also be Mech, so there's that). Thoughts?

### CODE REVIEW

**Comment by ThislsAUsername** [comment]
`GameAction.getUnitActionType()`
I don't like verbosity. I'm considering just making that change on `fixWally`, though. Thoughts?

File: res/map/Spann_Teamplay.map [hunk 1]
```diff
@@ -1,10 +0,0 @@
-  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE
-  SE  SE  GR  FR  RD  RD  BR  RD  RD  GR  FR 0FC  GR 0FC  SE
-  SE  GR  GR  FR  RD  FR  SE  GR  RD  CT  GR  GR 3HQ 3FC  SE
-  SE  CT  CT  RD  RD  SE  SE  SE  RD  GR  MT  GR  RD 3FC  SE
-  SE  GR  GR  RD  GR  SE  SE  SE  RD 0HQ  MT  CT  RD  GR  SE
-  SE 2FC  RD  RD 1HQ  SE  SE  FR  RD  RD  RD  RD  RD  CT  SE
-  SE 2FC 2HQ  GR  GR  GR  GR  FR  FR  GR  SE  SE  BR  SE  SE
-  SE  SE 1FC 1FC  GR  GR  GR  MT  GR  GR  SE  SE  GR  GR  SE
-  SE  RF  SE  SE  MT  CT  MT  FR  FC  GR  BR  GR  CT  CT  SE
-  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE  SE
```
Comment (ThislsAUsername) [inline res/map/Spann_Teamplay.map]: You clearly just have no taste.
It's like a fine wine, I tell you!

**Comment by ThislsAUsername** [comment]
I don't really want to do the tank renames, as that might cause undue confusion.

I guess we could do inf -> rifles and mech -> RPG, but we'd also have to update all the maps and that sounds like a pain.
I'm tempted to say "don't change it until we've got a reason to change it" but that just sounds like a way to increase technical debt.

I guess what I'm saying is I don't know lol

File: res/map/Spann_Teamplay.map (hunk 1 above)
Comment (ThatIsAUsername) [reply res/map/Spann_Teamplay.map]: I don't appreciate fine wines, either. Or escargot, if we want to add in that comparison.

**Comment by ThatIsAUsername** [comment]
> I guess what I'm saying is I don't know lol

We'll hold off on that one for now then.

> I don't like verbosity.

I don't have a problem with it, really, but I wouldn't object if you changed this.

**Comment by ThislsAUsername** [comment]
> I don't have a problem with it, really, but I wouldn't object if you changed this.

I'll just do it on Wally to avoid conflicts.

**Pull request merged**